examples/advanced/suit_update/Makefile.suit.custom

```diff
@@ -3,11 +3,6 @@ BINDIR_APP = $(CURDIR)/bin/$(BOARD)/$(APPLICATION)
 $(BINDIR_APP): $(CLEAN)
 	$(Q)mkdir -p $(BINDIR_APP)
 
-# Include to be able to use memoized
-include $(RIOTBASE)/makefiles/utils/variables.mk
-EPOCH = $(call memoized,EPOCH,$(shell date +%s))
-APP_VER ?= $(EPOCH)
```

Comment:
The `APP_VER` is used later on in this file.
When it is removed here you would expect that it comes form RIOT `Makefile.include`.
So this file would nedd to be included after RIOT Makefile.include.
But if this file is included after RIOT Makefile.include, then below `SUIT_CLIENT` and `SUIT_COAP_SERVER` are oerwritten by `suit.inc.mk`.
I think it is a bug that `examples/advanced/suit_update/Makefile.suit.custom` is included after `$(RIOTBASE)/Makefile.include`.